### ISSUE TITLE
Connect with Nextcloud Files

### ISSUE THREAD

**Issue by TtuxX** [hegocre/NextcloudPasswords#74]
### Checklist

- [X] I have checked and there are *no existing issues* - [open](https://github.com/hegocre/NextcloudPasswords/issues) or [closed](https://github.com/hegocre/NextcloudPasswords/issues?q=is%3Aissue+is%3Aclosed) - which request the same feature.
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.

### Feature description

I would love to be able to login very fast using Nextcloud Files, like this is the case on Nextcloud Deck android app

### Why do you want this feature?

For convenience and speed :)

**Comment by Saijin-Naib** [comment]
Yeah, I quite like that Account Add/Switch flow... would also make it easier to use my Work keychain/account as needed

**Comment by hegocre** [comment]
Hello @TtuxX 

I'm gonna take a look at this, but from what I have seen, it makes you use deprecated methods, which I don't like :(.

I think I'm gonna work on other issues (a lot this week), and revisit this after.

Cheers!

**Comment by TtuxX** [comment]
Nextcloud app login is becoming deprecated or unmaintained? 0.0

I understand then

Thank you very much for your work!
Side note: I would like to donate a bit but I am having issue with PayPal, would it be possible to add an alternative donation platform like https://www.buymeacoffee.com (or something as open and flexible as possible?)

**Comment by hegocre** [comment]
Hmm, not exactly. But the core methods used to login are deprecated, there is an issue from more than a year ago but it hasn't been resolved, which makes me think that the library is not greatly maintained, even though there is an update from a few months back. That's why I will focus on other issues and look more into it later.

About the second issue, I will take a look and tell you something back. Thanks!
